Documentation/specs/runtime-configuration-file.md

```diff
@@ -25,6 +25,10 @@ The files are both JSON files stored in UTF-8 encoding. Below are sample files.
     "runtimeConfig": {
         "gcServer": true,
         "gcConcurrent": false
+    },
+    "targetFramework": {
```

Comment:
That's conflicting with what I heard yesterday. Is that a recent change? I'm pretty sure I explicitly asked if it was in the editable ".config.json" file and heard that it was :)